docs/empower.rst

```diff
@@ -0,0 +1,47 @@
+Empower
+=======
+
+********
+Overview
+********
+
+The Empower class allows you to interact with the Empower API. Documentation for the Empower API can be found
+in their `GitHub <https://github.com/getempower/api-documentation/blob/master/README.md>`_ repo.
+
+.. note::
+    The Empower API only has a single endpoint to access all account data. As such, it has a very high overhead. This
+    connector employs caching in order to allow the user to specify the tables to extract without additional API calls.
+    You can disable caching as an argument when instantiating the class.
+
+==========
+Quickstart
+==========
+
```

Comment:
Typos: missing "as" in "as an environment variable"; also some confusion of singular key vs plural